submissions/tosinIshola/HTML/Ecommerce.html

```diff
@@ -17,40 +17,44 @@ <h1><strong>ALL THINGS FASHION (A.S.F)</strong></h1>
                 <li><a href="">OUR PRODUCT</a> </li>
                 <li><a href=""> CUSTOMER SERVICE</a> </li>
             </ul>
-        
+
 </nav>
 
 <hr>
 <h2>ABOUT US</h2>
 <p>
-    All Things Fashion (A..S.F) is an online store where different sellers can display and sell any item that has to do with fashion ranging from Cloths, Accesories, Hairs, Body Fitness and Enhancement, Skin Care, Makeup, Shoes and Bags e.t.c and buyers can purchase whatever they need from the store.
+    All Things Fashion (A.T.F) is an online store where different sellers can display and sell any item that has to do with fashion ranging from Cloths, Accesories, Hairs, Body Fitness and Enhancement, Skin Care, Makeup, Shoes and Bags e.t.c and buyers can purchase whatever they need from the store.
 </p>
 
 <hr>
+
 <h2>OUR PRODUCTS</h2>
 
-<h3><strong>Adult Female Mannequin</strong></h3>
+<h3>Adult Female Mannequin</h3>
 <img src="Images/dummy-2.jpg">
-<p><strong>PRICE:</strong><em>#28,000</em></p>
```

Comment:
The strong-tag could be within a p-tag because paragraphs are not strong by default. I was particular about having the strong-tag within a heading-tag.